packages/flutter_tools/lib/src/build_runner/build_script_generator.dart

```diff
@@ -1,4 +1,4 @@
-// Copyright 201 The Chromium Authors. All rights reserved.
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
This should probably be a different year.

Reply:
It's 2019